strongbox-commons/src/test/java/org/carlspring/strongbox/config/PropertiesPathResolverConfigTest.java

```diff
@@ -0,0 +1,112 @@
+package org.carlspring.strongbox.config;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.core.env.Environment;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.carlspring.strongbox.config.PropertiesPathResolverConfig.PropertiesPathResolver;
+import static org.carlspring.strongbox.config.PropertiesPathResolverConfig.PropertiesPathResolver.PREFIX_OVERRIDE_PROPERTY;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class PropertiesPathResolverConfigTest
+{
+    private static final String CURRENT_DIR = System.getProperty("user.dir");
+    private static final String CUSTOM_PATH_PROP = "strongbox.scary-t.file";
+    private static final String STRONGBOX_HOME_PROP = "strongbox.home";
+    private static final String STRONGBOX_HOME_DEFAULT = "/Users/keyboard-kitty/work/sbox";
```

Comment:
This won't work on other people's machines... Could you please fix it? Thanks!
(Also the path should be `strongbox` and not be abbreviated).

Reply:
These are mock paths, not real ones